src/ITILValidationTemplate.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2023 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+/**
+ * ValidationTemplate Class
+ **/
+class ITILValidationTemplate extends AbstractITILChildTemplate
+{
+    // From CommonDBTM
+    public $dohistory = true;
+
+    public static $rightname = 'itilvalidationtemplate';
+
+    public $can_be_translated = true;
+
+
+    public static function getTypeName($nb = 0)
+    {
+        return _n('Validation template', 'Validation templates', $nb);
+    }
+
+
+    public function getAdditionalFields()
+    {
+        $target   = new ITILValidationTemplate_Target();
+        $user_ids = array_map(function ($target) {
```

Comment:
It seems like `$users_id` and `$groups_id` are assigned but never used, is it normal ?